breadbox/breadbox/service/metadata.py

```diff
@@ -0,0 +1,71 @@
+import logging
+from typing import Any, Optional
+
+from breadbox.crud import dataset as dataset_crud
+from breadbox.crud import types as types_crud
+from breadbox.db.session import SessionWithUser
+from breadbox.schemas.custom_http_exception import ResourceNotFoundError
+from breadbox.models.dataset import MatrixDataset, TabularDataset
+
+from depmap_compute.slice import SliceQuery
+
+log = logging.getLogger(__name__)
+
+
+def get_tabular_dataset_metadata_annotations(
+    db: SessionWithUser, dataset: MatrixDataset, metadata_col_name: str,
```

Comment:
The typing here for dataset is `MatrixDataset` but the name says `get_tabular_dataset_metadata_annotations` that is confusing and makes me wonder if your tests are correct

Reply:
Oh weird it looks like I accidentally reviewed the outdated version of this file. Disregard the above comment